File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.py
import copy
def solution(num_list):
    answer = copy.deepcopy(num_list)
    prev = num_list[len(num_list) - 2]
    last = num_list[len(num_list) - 1]
    
    if prev < last:
        answer.append(last - prev)
    else:
        answer.append(last * 2)
        
    return answer

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[num_list.length+1];
        int a = 0;
        for(int i=0; i<num_list.length; i++){
            answer[i] += num_list[i];
            a++;
        }
        if(num_list[a-1] > num_list[a-2]){
            answer[a] = num_list[a-1] - num_list[a-2];
        }else{
            answer[a] = num_list[a-1]*2;
        }
        return answer;
    }
}